pkgs/by-name/ma/MaaAssistantArknights/package.nix

```diff
@@ -0,0 +1,139 @@
+# imitate https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=maa-assistant-arknights
+
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, opencv
+, onnxruntime
+, eigen
+, zlib
+, asio
+, libcpr
+, python3
+, android-tools
+, rustPlatform
+, makeWrapper
+, maaVersion ? "4.23.3"
+, maaSourceHash ? "sha256-A41kk1Xg9c/QZHOhyakcBULzsNTrsfnGYZ0df+MKWfc="
+, }:
+
+let
+
+  fastdeploy = stdenv.mkDerivation rec {
+
```

Comment:
Is It a good idea to also expose these two packages?

Reply:
These is a customized version of Fastdeploy by MAA org，not the original version.

MaaCore will be exposed after maa-cli release after a commit requested by me that makes maa-cli independent from MaaCore.

This pr needs lots of work to do, after upstreams improve MAA these days.